### PR TITLE
Start using experimental btcpy.

### PR DESCRIPTION
**Pull request by belovachap**
@peerchemist @saeveritt 

Here's a pass at using the experimental btcpy ( https://github.com/PeerAssets/btcpy/pull/6 ) with pypeerassets. Some more of our tests pass now that btcpy can operate on different network constants :)

### CODE REVIEW

File: pypeerassets/transactions.py [hunk 1]
```diff
@@ -4,6 +4,7 @@
 from math import ceil
 from time import time
 
+from btcpy.constants import Constants
```
Comment (peerchemist) [inline pypeerassets/transactions.py]: linter reports that this import is not used.

Comment (belovachap) [reply pypeerassets/transactions.py]: i'll double check it, i bet the linter is right :)

what linter/command do you run? i should probably use it too

Comment (peerchemist) [reply pypeerassets/transactions.py]: flake-8

**Comment by peerchemist** [comment]
@belovachap do we close this one?

**Comment by belovachap** [comment]
Yeah, let's close this pull request (along with https://github.com/PeerAssets/btcpy/pull/6) and pick up over in  https://github.com/PeerAssets/pypeerassets/pull/121

**Pull request closed without merging**